config/config.go

```diff
@@ -309,9 +306,9 @@ func NewNetworkAgentConfig(yamlPath string) (*AgentConfig, error) {
 		return nil, err
 	}
 
-	// (Re)configure the logging from our configuration, with the network tracer logfile
-	if err := NewLoggerLevel(cfg.LogLevel, cfg.NetworkTracerLogFile, cfg.LogToConsole); err != nil {
-		return nil, fmt.Errorf("failed to setup network-tracer logger: %s", err)
+	// (Re)configure the logging from our configuration, with the network tracer log file + config options
+	if err := setupLogger(loggerName, cfg.NetworkTracerLogFile, cfg); err != nil {
+		log.Errorf("failed to setup configured logger: %s", err)
```

Comment:
Should we return an error here? I can imagine it being very confusing debugging an agent which is running without a logger configured.

Reply:
The default agent is initialized to output to the console, but yeah I can see this being a good case for returning an error